pg_stat_kcache.c

```diff
@@ -100,6 +100,14 @@ typedef uint32 pgsk_queryid;
 #define ParallelLeaderBackendId ParallelMasterBackendId
 #endif
 
+/* ExecutorStart hook */
+#if PG_VERSION_NUM >= 180000
+#define EXEC_START_RET bool
```

Comment:
nit: you use spaces here but tab for the pg_qualstats versions.  Let's be consistent and use tabs everywhere.

Reply:
fixed

Reply:
thanks!